src/test/java/uk/gov/justice/digital/hmpps/cmd/api/service/PrisonServiceTest.kt

```diff
@@ -31,22 +32,26 @@ internal class PrisonServiceTest {
         fun `Should get Prisons`() {
             val prisonsStub = getValidPrisons()
             every { prisonRepository.findAll()} returns prisonsStub
-            assertThat(service.getAllPrisons()).hasSize(2)
-            assertThat(service.getAllPrisons()).contains(prison1)
+
+            val prisons = service.getAllPrisons()
+
+            verify { prisonRepository.findAll() }
+            confirmVerified(prisonRepository)
+
```

Comment:
The main value of these Mocked tests is to test how we interact with the components we've mocked, so here we're checking we've called the right repository method and then not interacted with the repository again. The lines below we know are going to pass because we set up what to return in line 34